### PR TITLE
Add missing bullet point to staging-use-cases.md

### PR DESCRIPTION
**Pull request by astrieanna**
**What this PR does / why we need it**:
One of the sets of use-cases doesn't have a bullet point. There's only one now, but it should be bulleted list like all the other sections.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: docs/staging-use-cases.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Multitenancy Management
 - 🔭 As an operator, given I have created a Kubernetes-conformant cluster without ClusterAPI, I want to use ClusterAPI to manage it. In order to do so, I need to know the requirements for adopting/importing this cluster in terms of required CRD’s and operators (e.g Machine and Cluster objects).
 
 ### Multitenancy Management
-🔭 As an operator, given I have a management cluster and a workload cluster, I want to setup roles, role bindings, users, and usage quotas on my workload cluster.
+- 🔭 As an operator, given I have a management cluster and a workload cluster, I want to setup roles, role bindings, users, and usage quotas on my workload cluster.
```
Comment (vikaschoudhary16) [inline docs/staging-use-cases.md]: Can you please remove extra comma as well. s/users, and/users and

Comment (detiber) [reply docs/staging-use-cases.md]: I don't think it's worth it to hold this up over use of an Oxford comma.

Comment (ncdc) [reply docs/staging-use-cases.md]: We should be using Oxford commas

**Pull request merged**